crates/cheatcodes/spec/src/function.rs

```diff
@@ -31,6 +31,8 @@ pub struct Function<'a> {
     pub selector: &'a str,
     /// The 4-byte function selector as a byte array.
     pub selector_bytes: [u8; 4],
+    /// The replacement function (if any) for deprecated cheatcodes.
+    pub replacement: &'a str,
```

Comment:
This should go in `Deprecated(str)`

Reply:
changed in https://github.com/foundry-rs/foundry/pull/8883/commits/877d6fb4ca78d63fa53aabe955847102eb2b7e04 thank you for clarifying how should be done

Reply:
Even better! Neat